CMSgov/easi-app
# [EASI-3017] Notify CEDAR by email on role modifications

# EASI-3017

## Changes and Description

- Calculates metadata about the roleadd/roledelete requests for use in an email template
- Uses calculated metadata to send an email to the configured account (env var `CEDAR_EMAIL_ADDRESS`)

## Other Notes

- I didn't add email tests for this in favor of just manually testing with `cmd/test_email_templates`. This code isn't likely to live for very long, and the tests (alongside the ones written) just aren't worth the time right now.

## How to test this change

- `scripts/dev up:backend`
- `go run cmd/test_email_templates/main.go`
- Open http://localhost:1080
- Check email content!

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
